nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -35,8 +35,15 @@ with lib;
   # Enable sound in graphical iso's.
   hardware.pulseaudio.enable = true;
 
-  # Spice guest additions
+  # VM guest additions to improve host-guest interaction
   services.spice-vdagentd.enable = true;
+  services.qemuGuest.enable = true;
+  virtualisation.vmware.guest.enable = true;
```

Comment:
This is causing issues due to `virtualization.vmware.guest.enable` setting `videoDrivers = mkOverride 50 [ "vmware" ];
Only happens in the kde iso since gnome uses wayland
Perhaps the mkOverride should be removed

Reply:
https://github.com/NixOS/nixpkgs/pull/175445